shared/test/integration/tbpl-manifest.json

```diff
@@ -33,8 +33,6 @@
     "apps/settings/test/marionette/tests/app_permission_settings_test.js": "Bug 1004348 - Disable intermittent failing test, manipulate app permissions set geolocation of first app to Grant",
     "apps/settings/test/marionette/tests/hotspot_wifi_settings_test.js": "Bug 1006527 - Disable intermittent failing travis test, manipulate hotspot wifi settings resets settings for second load",
     "apps/settings/test/marionette/tests/message_settings_test.js": "",
-    "apps/sms/test/marionette/composer_test.js": "Bug 1121766 - TEST-UNEXPECTED-FAIL | Messages Composer Messages Composer Test Suite Message char counter and MMS label",
```

Comment:
Sorry, why do we enable `composer_test.js` here? Does the patch fix that issue? But I saw some failures of running `composer_test.js` in [1]. And it is also an intermittent failures.

[1]: https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=aec5a5ea89dc